sound/soc/sof/sof-priv.h

```diff
@@ -54,6 +54,16 @@ extern int sof_core_debug;
 	(IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_ENABLE_DEBUGFS_CACHE) || \
 	 IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST))
 
+/* DSP power state */
+enum sof_dsp_power_states {
+	SOF_DSP_D0,
```

Comment:
Minor nitpick, I would prefix with SOF_DSP_PM_

Reply:
that sounds indeed better. @ranj063 can you update the PR with those defines and I'll merge to trigger actual validation. Thanks!

Reply:
fixed now @plbossart